tests/behat/features/bootstrap/LibContext.php

```diff
@@ -20,1600 +20,1292 @@
+     *
+     * @param string $path
+     *   The path to navigate to.
      */
-    $link = $this->drupalContext->getSession()->getPage()->findLink('Brugerprofil');
-    if (!$link) {
-      throw new Exception("Couldn't find link to user profile on the users page");
+    public function gotoPage($path)
+    {
+        $this->minkContext->visit($path);
+    }
+
+    /**
+     * Go to the search page.
+     *
+     * @param string $string
+     *   String to search for.
+     *
+     * @Given I have searched for :string
+     *
+     * @throws Exception
+     *   In case of error.
+     */
+    public function gotoSearchPage($string)
+    {
+        // First we try to translate the argument, to see if there's anything
+        // we should pick out first.
+        $searchString = $this->translateArgument($string);
+        $this->logMsg(
+            ($this->searchPage->getVerboseSearchResult() == "on"),
+            "Searches for " . urlencode($searchString) . "\n"
+        );
+        $this->lastSearchString = $searchString;
+        $this->gotoPage('/search/ting/' . urlencode($searchString));
+        $this->waitForPage();
     }
-    $this->user = $user;
-  }
 
 
-  /**
-   * Log a user in.
-   *
-   * @throws Exception
-   *   When errors happens.
-   */
-  public function login() {
-
-    if (!$this->drupalUser) {
-      throw new Exception('Tried to login without a user.');
+    /**
+     * Implements step to search for data and put it into a file as known data.
+     *
+     * @param string $mfile
+     *    Filename basis for the two files being created, appended with _pos.mat and _neg.mat.
+     * @param string $relation
+     *    The relation we are looking for, in opensearch terminology, f.ex. dbcaddi:hasCreatorDescription.
+     *
+     * @Given I create files :mfile from opensearch on relation :relation
+     *
+     * @throws ErrorException
+     *   In case of error in calling the opensearch service.
+     */
+    public function ICreateFilesForRelation($mfile, $relation)
+    {
+        for ($i = 1; $i < 12000; $i = $i + 50) {
+            $this->ICreateFilesForRelationChunk($mfile, $relation, $i);
+        }
     }
 
-    // It's nice to know in the log who we log in with.
-    $this->logMsg(($this->verbose->loginInfo == "on"), "Attempts logging in with user: " . $this->drupalUser->name . "\n");
+    /**
+     * Helper function for ICreateFilesForRelation.
+     *
+     * @param string $mfile
+     *    Filename base for the two files being created.
+     * @param string $relation
+     *    The relation we are looking for.
+     * @param int $start
+     *    The starting point for the service retrieval.
+     *
+     * @throws ErrorException
+     *    In case of fetching from the search fails.
+     */
+    private function ICreateFilesForRelationChunk($mfile, $relation, $start)
+    {
+        // Set up the search as URL.
+        $url = "https://oss-services.dbc.dk/opensearch/4.5/";
```

Comment:
Note that this url is deprecated. Latest is https://oss-services.dbc.dk/opensearch/5.0/